qbatch

```diff
@@ -170,4 +202,8 @@ if __name__ == "__main__":
         os.chmod(script, os.stat(script).st_mode | stat.S_IXUSR)
         if dry_run:
             continue
-        subprocess.call(['qsub', script])
+        if system == 'sge' or system == 'pbs':
+            subprocess.call(['qsub', script])
+        else:
+            print "Launching Local Jobscript, saving output to logdir/{}.log".format(job_name)
```

Comment:
Right, i guess what I was saying was to *not* handling logging at all in the local mode, and leave that up to the user to do with redirection if they want it. 

I see the utility in logging, of course, but I want to keep away from adding too many option dependencies (i.e. `--logdir` only works when running in `local` mode)... Although, wait, I suppose if you make `--logdir` setup logging for the schedulers also, then it isn't confusing.